src/test/java/com/epam/poc/homePage/HomePageTest.java

```diff
@@ -28,7 +55,12 @@ public class HomePageTest extends BaseTest {
     public void beforeMethod() {
         homePage = new HomePageObject(driver);
         headerPage = new HeaderPageObject(driver);
+        homePage.closeHomePagePopup();
+    }
 
+    @AfterMethod
+    public void afterMethod() {
+        goToHomePage(driver);
         homePage.closeHomePagePopup();
```

Comment:
the function closeHomePagePopup() was already existed in @BeforeMethod, you can remove this command line in @AfterMethod

Reply:
the two calls make two different purposes, please check again